test/data/matpower/case5_pwlc.m

```diff
@@ -27,11 +27,11 @@
 %% generator cost data
 %	1	startup	shutdown	n	x1	y1	...	xn	yn
 mpc.gencost = [
-	1	0.0	0.0	4	 -0.1	   0.0	  0.0	   0.0	  0.1	   0.0	  0.2	   0.0;  % tests zero cost
-	1	0.0	0.0	4	  0.0	 841.0	 20.0	 841.0	 100.0	 841.0	150.0	 841.0;  % tests constant cost
-	1	0.0	0.0	4	170.0	4772.0	231.0	6203.0	293.0	7855.0	355.0	9738.0;
-	1	0.0	0.0	4	 22.0	1122.0	 33.0	1417.0	 44.0	1742.0	 55.0	2075.0;
-	1	0.0	0.0	4	  7.0	 897.0	  8.0	1187.0	  9.0	1479.0	 10.0	1791.0;
+	1	0.0	0.0	4	 -0.1	   0.0	  0.0	   0.0	  0.1	   0.0	  0.2	   0.0	   2.0;  % tests zero cost
+	1	0.0	0.0	4	  0.0	 841.0	 20.0	 841.0	 100.0	 841.0	150.0	 841.0	   2.0;  % tests constant cost
+	1	0.0	0.0	4	170.0	4772.0	231.0	6203.0	293.0	7855.0	355.0	9738.0	   2.0;
+	1	0.0	0.0	4	 22.0	1122.0	 33.0	1417.0	 44.0	1742.0	 55.0	2075.0	   2.0;
+	1	0.0	0.0	4	  7.0	 897.0	  8.0	1187.0	  9.0	1479.0	 10.0	1791.0	   2.0;
```

Comment:
This is an interesting case becouse the x_5 value does have a matching y_5 value.  Is this an expectable Matpower file?  If so, it is fine as is.  If not, just add a y_5 value to make it valid.

Reply:
Matpower does not seem to mind this.